File: datasets/IR-Plag/convertai-original-java-rest-python/case-05/plagiarized/L4/04/Kasus5.py
import sys

def reverse(number):
    print(number % 10, end='')

def main():
    print("Enter an integer: ", end='')
    number = int(input())
    
    while number > 0:
        reverse(number)
        number //= 10
    print()

if __name__ == "__main__":
    main()

File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-05/plagiarized/L6/04/Kasus5.kt
package IR

import java.util.*

/*
* To change this license header, choose License Headers in Project Properties.
* To change this template file, choose Tools | Templates
* and open the template in the editor.
*/

/**
 *
 * @author C166AB8FBBBE55174E16A67530426A8A 19011F43894F06E2BA0CA74061427469
 */
object Kasus5 {
    fun reverse(number: Int) {
        var number = number
        if (number > 0) {
            print(number % 10)
            reverse(10.let { number /= it; number })
        }
    }

    @JvmStatic
    fun main(args: Array<String>) {
        print("Enter an integer: ")
        val input = Scanner(System.`in`)
        val number = input.nextInt()

        reverse(number)
        println("")
    }
}
